src/Uno.UI/Controls/NativeFramePresenter.iOS.cs

```diff
@@ -122,8 +122,21 @@ private void InitializeController(Frame frame)
 
 			if (_frame.Content is Page startPage)
 			{
-				var viewController = new PageViewController(startPage);
-				NavigationController.PushViewController(viewController, false);
+				// When the frame already has content, we add a NavigationRequest in the PageViewController's AssociatedRequests.
+				// Not doing this results in log errors.
```

Comment:
can you detail which errors ?

Reply:
done :)

Reply:
It's likely something we can add tests for, also.

Reply:
The only difference in behavior is the content of the log output. It's a really minor fix.

Reply:
Do you require a test for log output?

Reply:
Not really, if it's just the logs then let's leave it as that.